recipes/containers/lighttpd/index.html

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
Honestly? :astonished: :wink:

Reply:
I've copied it from the lighttpd 404 page... :laughing: